src/lsp/cobol_parser/text_tokenizer.ml

```diff
@@ -187,6 +315,51 @@ let preproc_n_combine_tokens ~source_format =
         consume_comment ~loc:(hd l) ~revtoks:[] ~at_end
           Comment_entry acc suff
     in
+    let function_specifier = function
```

Comment:
Same. Maybe the changes need to be handled in the same way as picture strings in the other function below.